### ISSUE TITLE
Lots of broken files to test

### ISSUE THREAD

**Issue by OmgImAlexis** [guessit-io/guessit#399]
Here's a tree of a bunch of releases from the last 2 or so weeks, most of which break in guessit one way or another.

```
.
├── 2.Broke.Girls.S06E12.1080p.Amazon.WEBRip.DD+.5.1.x264-TrollHD
│   └── 3HwkquVdRcnPo2sfJgtBfvbqDVdxsVUt.mkv
├── Between.2015.S01E02.1080p.AC3.5.1.WEB-DL
│   ├── Bestandsnamen\ omdraaien
│   │   ├── Online\ tekst\ omdraaien.url
│   │   ├── QoQ\ Reverse\ for\ Macintosh.url
│   │   ├── QoQ\ Reverse\ for\ Windows.url
│   │   └── Reverse\ Rename\ File\ Script.url
│   ├── Flyer.jpg
│   ├── LD-BEW.1.5.3CA.p0801.20E10S.5102.neewteB.mkv
│   └── WickedReaction.url
├── Blindspot.S02E10.Nor.I.Nigel.AKA.Leg.in.Iron.AMZN.1080p.WEBRip.DD5.1.x264-DRACULA
│   └── 0fMr4NHfdy9g4.mkv
├── Blindspot.S02E10.Nor.I.Nigel.AKA.Leg.in.Iron.AMZN.1080p.WEBRip.DD5.1.x264-DRACULA.1
│   └── 0fMr4NHfdy9g4.mkv
├── Bones.S12E01.720p.HDTV.x264-AVS
│   └── nsW2WMcZvKfTGf1HsNdhyD3DZSZhzNVeE2pmSwk.mkv
├── Doctor.Who.2005.S06E02.720p.HDTV.DD5.1.x264
│   └── EnSqO84GZUjaFNdpJ5HY.mkv
├── Doctor.Who.2005.S06E03.720p.HDTV.DD5.1.x264
│   └── NSi1cnZ0rFBaObfrmsGP.mkv
├── Doctor.Who.2005.S06E06.720p.HDTV.DD5.1.x264
│   └── wRodrwqhgava7xd8qR7d.mkv
├── Finding.Carter.S02E01.Love.the.Way.You.Lie.1080p.WEB-DL.AAC2.0.H.264-NL
│   ├── Kijk\ hier\ voor\ nog\ meer\ spots.URL
│   ├── LN-462.H.0.2CAA.LD-BEW.p0801.eiL.uoY.yaW.eht.evoL.10E20S.retraC.gnidniF.mkv
│   ├── LN-462.H.0.2CAA.LD-BEW.p0801.eiL.uoY.yaW.eht.evoL.10E20S.retraC.gnidniF.srt
│   ├── Place2home\ Torrents.url
│   ├── Usenet\ Provider\ Newsconnection.nl.url
│   └── flyer\ Newsconnection.nl.jpg
├── Finding.Carter.S02E03.We.are.never.ever.1080p.WEB-DL.AAC2.0.H.264.NL
│   ├── Kijk\ hier\ voor\ nog\ meer\ spots.URL
│   ├── LN.462.H.0.2CAA.LD-BEW.p0801.reve.reven.era.eW.30E20S.retraC.gnidniF.mkv
│   ├── LN.462.H.0.2CAA.LD-BEW.p0801.reve.reven.era.eW.30E20S.retraC.gnidniF.srt
│   ├── Place2home\ Torrents.url
│   ├── Usenet\ Provider\ Newsconnection.nl.url
│   └── flyer\ Newsconnection.nl.jpg
├── Finding.Carter.S02E06.Stay.With.Me.1080p.WEB-DL.AAC2.0.H.264-QUEENS
│   ├── Kijk\ hier\ voor\ nog\ meer\ spots.URL
│   ├── LN-462.H.0.2CAA.LD-BEW.p0801.eM.htiW.yatS.60E20S.retraC.gnidniF.mkv
│   ├── LN-462.H.0.2CAA.LD-BEW.p0801.eM.htiW.yatS.60E20S.retraC.gnidniF.srt
│   ├── Place2home\ Torrents.url
│   ├── Usenet\ Provider\ Newsconnection.nl.url
│   └── flyer\ Newsconnection.nl.jpg
├── Finding.Carter.S02E09.I.Knew.You.Were.Trouble.1080p.WEB-DL.AAC2.0.H.264-NL
│   ├── Kijk\ hier\ voor\ nog\ meer\ spots.URL
│   ├── LN-462.H.0.2CAA.LD-BEW.p0801.elbuorT.ereW.uoY.wenK.I.90E20S.retraC.gnidniF.mkv
│   ├── LN-462.H.0.2CAA.LD-BEW.p0801.elbuorT.ereW.uoY.wenK.I.90E20S.retraC.gnidniF.srt
│   ├── Place2home\ Torrents.url
│   ├── Usenet\ Provider\ Newsconnection.nl.url
│   └── flyer\ Newsconnection.nl.jpg
├── Finding.Carter.S02E12.Im.Not.the.Only.One.1080p.WEB-DL.AAC2.0.H.264-NL.1
│   ├── )1(\ LN-462.H.0.2CAA.LD-BEW.p0801.enO.ylnO.eht.toN.mI.21E20S.retraC.gnidniF.mkv
│   ├── Kijk\ hier\ voor\ nog\ meer\ spots.URL
│   ├── Place2home\ Torrents.url
│   ├── Usenet\ Provider\ Newsconnection.nl.url
│   └── flyer\ Newsconnection.nl.jpg
├── Finding.Carter.S02E16.The.Sound.And.The.Fury.1080p.WEB-DL.AAC2.0.H.264-QUEENS
│   └── YUoV6HlqNz7slEKXAdRwRCBIlV.mkv
├── Fresh.Off.the.Boat.S01E07.Showdown.At.The.Golden.Saddle.1080p.WEB-DL.DD5.1.H.264-COO7
│   └── 160505_14.mkv
├── Fresh.Off.the.Boat.S01E09.License.To.Sell.1080p.WEB-DL.DD5.1.H.264-COO7
│   └── 160505_18.mkv
├── Fresh.Off.the.Boat.S01E10.Blind.Spot.1080p.WEB-DL.DD5.1.H.264-COO7
│   └── 160505_20.mkv
├── Fresh.Off.the.Boat.S01E11.Very.Superstitious.1080p.WEB-DL.DD5.1.H.264-COO7
│   └── 160505_22.mkv
├── Fresh.Off.the.Boat.S01E12.Dribbling.Tiger.Bounce.Pass.Dragon.1080p.WEB-DL.DD5.1.H.264-COO7
│   └── 160505_24.mkv
├── Hannibal.S03E10.1080p.WEB-DL.DD5.1.H.264-PSYPHER
│   ├── 150806_05.mkv
│   ├── 150806_05.nfo
│   ├── 150806_05.nzb
│   └── 150806_05.srr
├── Hannibal.S03E10.1080p.WEB-DL.DD5.1.H.264-PSYPHER.1
│   ├── 150806_05.mkv
│   ├── 150806_05.nfo
│   ├── 150806_05.nzb
│   └── 150806_05.srr
├── House.S04E14.1080p.WEB-DL.DD5.1.H.264
│   ├── Backup_73255S04-14.mkv
│   ├── Backup_73255S04-14.mkv.par2
│   └── Backup_73255S04-14.nzb
├── Its.Always.Sunny.in.Philadelphia.S11E05.Mac.and.Dennis.Move.To.The.Suburbs.1080p.WEB-DL.AAC2.0.H.264-OOSH
│   ├── 160126_08.mkv
│   ├── 160126_08.nfo
│   ├── 160126_08.nzb
│   └── 160126_08.srr
├── Mr.Robot.S02E12.1080p.WEB-DL.DD5.1-NL.Subs-Het.Robot.Team.OYM
│   └── sbuS\ LN-1.5DD\ LD-BEW\ p0801\ 21E20S\ toboR\ .rM.mkv
├── Murder.in.the.First.S03E03.Black.and.Blue.1080p.WEB-DL.DD5.1.H.264-VISUM
│   ├── 160908_06.mkv
│   ├── 160908_06.nfo
│   ├── 160908_06.nzb
│   └── 160908_06.srr
├── Notorious.S01E03.Friends.and.Other.Strangers.1080p.WEB-DL.DD5.1.H.264-AG
│   ├── 86135581.txt
│   └── vRmaU2VbgiJVxPirgO07FRnxTS3ZbEz1rq4.mkv
├── Orange.Is.the.New.Black.S02E05.1080p.WEBRip.x264.DD5.1-AbsinthREPOST
│   └── lllll8237564lllll
│       └── Orange.Is.The.New.Black.S02E05.1080p.WEBRip.x264.DD5.1-Absinth.mkv
├── Penn.&.Teller.Fool.Us.S03E13.720p.HDTV.x264-w4f
│   └── eSYHHYGo2HuqNvb6vMF5Of4Ky05yw46SV.mkv
├── Proof.2015.S01E03.1080p.WEB-DL.DD5.1.H.264-KINGS
│   └── kAyUDeygUHGF0TKgsAgpZ.mkv
├── Proof.2015.S01E05.1080p.WEB-DL.DD5.1.H.264-KINGS
│   ├── rhzsvaZe0evwItfCiGFhg2vs9c2MVNHdxcWM.mkv
│   └── rhzsvaZe0evwItfCiGFhg2vs9c2MVNHdxcWM.sfv
├── Proof.2015.S01E06.1080p.WEB-DL.DD5.1.H.264-KINGS
│   ├── kP5SNn5sTYlEFCUV.mkv
│   └── kP5SNn5sTYlEFCUV.sfv
├── Proof.2015.S01E08.1080p.WEB-DL.DD5.1.H.264-KINGS
│   └── tCU7EYQJBGX6o19uSeuJAF61VnSN5zsid3VBbiP.mkv
├── Proof.2015.S01E09.1080p.WEB-DL.DD5.1.H.264-KINGS
│   ├── I3YvWqzurZg7Plb1OWE5ekZyimoL95IVCYPTYFSNlfRqRSlpTThx2qd.mkv
│   └── I3YvWqzurZg7Plb1OWE5ekZyimoL95IVCYPTYFSNlfRqRSlpTThx2qd.sfv
├── Proof.2015.S01E10.1080p.WEB-DL.DD5.1.H.264-KINGS
│   ├── RbMWnZuOhAGkKkr9s1fDejdCdE.mkv
│   └── RbMWnZuOhAGkKkr9s1fDejdCdE.sfv
├── Public.Morals.S01E08.No.Crazies.On.The.Street.1080p.WEB-DL.DD5.1.H.264-CTRLHD
│   ├── 59915175.txt
│   └── wnM74Y0Wprqr8MQwVJ9P.mkv
The.Big.Bang.Theory.S04E11.The.Justice.League.Recombination.2010.BluRay.720p.x264.DXVA.DTS-de42
│   ├── 68340583.txt
│   └── gF52wxWkbbX.mkv
├── The.Vampire.Diaries.S05E13.720p.WEB-DL.DD5.1.H.264-KiNGS
│   └── S07BPWb4k1i71Y.mkv
├── The.Vampire.Diaries.S07E05.1080p.WEB-DL.DD5.1.H.264-DRACULA
│   ├── 151101_10.mkv
│   ├── 151101_10.nfo
│   ├── 151101_10.nzb
│   └── 151101_10.srr
├── We.Bare.Bears.S01E14.Brother.Up.1080p.WEB-DL.AAC2.0.H.264-TVSmash
│   ├── 36236326.txt
│   └── mxNMuJWeO7PUWCMEwqKSsS6D8Vs9S6V3PHD.mkv
```

**Comment by Toilal** [comment]
Can you confirm that backslashes are only escape characters ?

**Comment by OmgImAlexis** [comment]
@Toilal yep just escape chars from the `tree` command.

**Comment by OmgImAlexis** [comment]
```
For: Bones.S12E02.The.Brain.In.The.Bot.1080p.WEB-DL.DD5.1.H.264-R2D2/161219_06.mkv
GuessIt found: {
    "title": "Bones", 
    "season": 12, 
    "episode_title": "The Brain In The Bot", 
    "screen_size": "1080p", 
    "format": "WEB-DL", 
    "audio_codec": "DolbyDigital", 
    "audio_channels": "5.1", 
    "video_codec": "h264", 
    "release_group": "R2D2", 
    "episode": 6, 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}
```

**Comment by ratoaq2** [comment]
Fixed by #405
